pkg/juicefs/juicefs.go

```diff
@@ -32,6 +31,7 @@ import (
 	"time"
 
 	"github.com/juicedata/juicefs-csi-driver/pkg/util"
+	"github.com/pkg/errors"
```

Comment:
？

Reply:
The `github.com/pkg/errors` has more features than `errors` like [`Wrap`](https://pkg.go.dev/github.com/pkg/errors#Wrap) function.